includes/admin/class-wc-admin-help.php

```diff
@@ -42,19 +42,13 @@ public function add_tabs() {
 				'title'   => __( 'Help &amp; Support', 'woocommerce' ),
 				'content' =>
 					'<h2>' . __( 'Help &amp; Support', 'woocommerce' ) . '</h2>' .
-					'<p>' . sprintf(
-						/* translators: %s: Documentation URL */
-						__( 'Should you need help understanding, using, or extending WooCommerce, <a href="%s">please read our documentation</a>. You will find all kinds of resources including snippets, tutorials and much more.', 'woocommerce' ),
-						'https://docs.woocommerce.com/documentation/plugins/woocommerce/?utm_source=helptab&utm_medium=product&utm_content=docs&utm_campaign=woocommerceplugin'
-					) . '</p>' .
 					'<p>' . sprintf(
 						/* translators: %s: Forum URL */
-						__( 'For further assistance with WooCommerce core you can use the <a href="%1$s">community forum</a>. If you need help with premium extensions sold by WooCommerce, please <a href="%2$s">use our helpdesk</a>.', 'woocommerce' ),
-						'https://wordpress.org/support/plugin/woocommerce',
-						'https://woocommerce.com/my-account/tickets/?utm_source=helptab&utm_medium=product&utm_content=tickets&utm_campaign=woocommerceplugin'
+						__( 'For further assistance with Classic Commerce you can use the <a href="%1$s">ClassicPress community forum</a>.', 'woocommerce' ),
+						'https://forums.classicpress.net/'
```

Comment:
__( 'For further assistance with Classic Commerce you can use the <a href="%1$s">Plugin Github issues section</a>.', 'woocommerce' ),
						'https://github.com/ClassicPress-research/classic-commerce/issues'